.github/workflows/tests.yaml

```
@@ -25,7 +25,7 @@ jobs:
         run: sudo apt-get update -q
       - uses: actions/setup-node@v3
```

already merged but this could be bumped too

```suggestion
      - uses: actions/setup-node@v4
```

---

I activated dependabot on the repo (don't know why it was deactivated but configured at the same time), to open the PR #12 for this bump.